rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncher.xtend

```diff
@@ -69,6 +77,36 @@ public class TclMavenLauncher implements TclLauncher {
 
 	}
 
+	def checkMavenVersion(IProject project, IProgressMonitor monitor) {
+		val versionOut = new ByteArrayOutputStream()
+		val infoResult = mavenExecutor.executeInNewJvm("-version", project.location.toOSString, "", monitor, versionOut,
+			false)
+		if (infoResult != IStatus.OK) {
+			logger.error("Error during determine maven version")
+			return false
+		}
+		val versionInfoScanner = new Scanner(versionOut.toString).useDelimiter("\n");
+		
+		while (versionInfoScanner.hasNext) {
+			val line = versionInfoScanner.next()
+			if (line.startsWith("Apache Maven ")) {
+				var version = line.substring("Apache Maven ".length)
+				logger.info("Maven Version: {} ", version)
+				if (!version.startsWith("3.2.5")) {
+					version = version.substring(0, version.indexOf(' '))
+					if (JOptionPane.showConfirmDialog(null, "Wrong maven-version '" + version +
```

Comment:
The other dialogs are swt based dialogs. As a user i would expect for every dialog the same style.